src/scss/sections/_playground.scss

```diff
@@ -0,0 +1,43 @@
+.col_sec_des {
+  line-height: $line-height-xl;
+
+  @include media-breakpoint-up(lg) {
+    font-size: $h4-font-size-lg;
+  }
+}
+
+.collection-list {
+  text-align: center;
+
+  figcaption {
+    bottom: 50%;
+    transform: translateY(50%);
+    width: 100%;
+    position: absolute;
+    margin: 0 auto;
+    left: 0;
+    right: 0;
+    padding: 0;
+    top: 0;
+
+    @include media-breakpoint-up(lg) {
+      top: -50%;
```

Comment:
please manage the responsiveness in the same way. by default you use bottom and for desktop you use top. you can use the same value for position